File: 1erdepartamental/python/7.creacion_dato/main.py
"""
Programa: Creación de Datos
Demuestra tipos de datos personalizados con __str__
"""

from core.tipos import Persona, Auto

if __name__ == "__main__":
    p = Persona("Hugo", "Dominguez", 19)
    print(p)

    a = Auto("Honda", "Civic", 350000)
    print(a)

File: 1erdepartamental/c++/7.creacion_dato/main.cpp
#include <iostream>
#include "core/Persona.h"
#include "core/Auto.h"

int main() {
    Persona p("Hugo", "Dominguez", "Lopez", 'M', 19);
    std::cout << p << std::endl << std::endl;
    
    Auto a("Honda", "Civic", 350000, 2022);
    std::cout << a << std::endl;
    
    return 0;
}
